src/engraving/libmscore/textbase.cpp

```diff
@@ -961,7 +961,7 @@ mu::draw::Font TextFragment::font(const TextBase* t) const
     QString family;
     if (format.fontFamily() == "ScoreText") {
         if (t->parent() && t->isDynamic()) {
-            family = t->score()->styleSt(Sid::MusicalSymbolFont);
+            family = t->score()->scoreFont()->fontByName(t->score()->styleSt(Sid::MusicalSymbolFont))->family();
```

Comment:
Why this change?

Reply:
This fixes the issue where Emmentaler and Gonville simply displayed Bravura instead. Internally, Emmentaler is called "mscore" and Gonville is called "Gootville". This change maps the fonts to the correct values.

You could alternatively hardcode in the mappings, but that would make it harder to maintain later as the `ScoreFont` class should be the single source of truth for the fonts.

Reply:
OK, so as a hack for 3,x (which is apparently lacking that `ScoreFont::fontByName()` we could use this, right?
```
                  family = t->score()->styleSt(Sid::MusicalSymbolFont);
                  if (family == "Emmentaler")
                        family = "MScore";
                  else if (family == "Gonville")
                        family = "Gootville";
```

Reply:
That should work.

Reply:
Well, unfortunaly it dosn't really, the vtest for Dynamics are just way too large and not in the ScoreFont style, most obvious for MuseJazz, but the same for Emmentaler, Gonville, Bravura.
I could do a plain PR of this one here on top of the latest 3.x, would that help?